src/resource/solidDataset.ts

```diff
@@ -1202,6 +1202,19 @@ export async function getWellKnownSolid(
   > = internal_defaultFetchOptions
 ): Promise<SolidDataset & WithServerResourceInfo> {
   const urlString = internal_toIriString(url);
+
+  // Try to fetch the well-known solid dataset from the server's root (ESS 2.0 behaviour)
```

Comment:
This is a comment out of scope for the current PR, but do you think it would be worth documenting systematically, for each function of the public API, if it is a function built against the Solid spec or a particular implementation ? Something like a custom TSDoc tag for instance.

Reply:
Custom TSDoc tag sounds interesting, but I'm not sure it would be very appropriate inside of functions or how standard it is (a quick search [yielded an issue](https://github.com/microsoft/tsdoc/issues/21), but nothing on [the docs](https://tsdoc.org)).

I see however that there is an [experimental tag](https://tsdoc.org/pages/tags/experimental/) that I would recommend using for everything that is not standard. I also think it would be good to [tag as alpha](https://tsdoc.org/pages/tags/alpha/) everything that is not in a versionned standard ([at this point in time everything but Solid Protocol](https://solidproject.org/TR/)).

Furthermore, the [see tag](https://tsdoc.org/pages/tags/see/) seems appropriate for spec links and the [label tag](https://tsdoc.org/pages/tags/label/) is probably the best fit for implementation specific functions.

Reply:
(I removed this version pointer for now)